Source/DataSources/BillboardVisualizer.js

```diff
@@ -162,6 +164,35 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
+     * @returns {BoundingSphereState} BoundingSphereState.DONE if the result contains the bounding sphere,
+     *                       BoundingSphereState.PENDING if the result is still being computed, or
+     *                       BoundingSphereState.FAILED if the entity has no visualization in the current scene.
+     */
+    BillboardVisualizer.prototype.getBoundingSphere = function(entity, result) {
```

Comment:
Okay, even though I put up a bunch of opposition, I want ahead and marked `BoundingSphereState` and related `getBoundingSphere` calls in the visualizers as private for now.  I still think they ultimately need to be public, but we might want to tweak the names and let them mature for a while before exposing them.  This way if we have issues with new functionality in 1.6, we won't have to worry about breaking changes.